src/translations/en.json

```diff
@@ -919,6 +919,29 @@
           "working": "Please wait",
           "unknown_error": "Something went wrong",
           "providers": {
+            "command_line": {
+              "step": {
+                "init": {
+                  "data": {
+                    "username": "Username",
+                    "password": "Password"
+                  }
+                },
+                "mfa": {
+                  "data": {
+                    "code": "Two-factor Authentication Code"
```

Comment:
Please use `[%key:...%]` to reference other exist resource, for example:

```
"code": [%key:ui::panel::page-authorize::form::providers::homeassistant::step::mfa::data::code%]
```

Reply:
Can't I just reference the whole mfa section?

    "mfa": [%key:ui::panel::page-authorize::form::providers::homeassistant::step::mfa%]

Reply:
Or even

    "command_line": { [%key:ui::panel::page-authorize::form::providers::homeassistant%],

since there's no difference at all.

Reply:
No, it won't work.

Reply:
Too bad.